tools/packaging/common/istio-iptables.sh

```diff
@@ -48,6 +48,7 @@ function usage() {
   echo '      outbound traffic (i.e. "*") is being redirected (default to $ISTIO_SERVICE_EXCLUDE_CIDR).'
   echo '  -k: Comma separated list of virtual interfaces whose inbound traffic (from VM)'
   echo '      will be treated as outbound (optional)'
+  echo '  -t: Unit testing, only functions are loaded and no other instructions are executed.'
```

Comment:
Just use an environment variable - it's not something to expose to users.

Reply:
it is a possibility but that introduces more than 1 control of the script, right now everything runs around `while getops` and it kind of make sense, adding another control of the script flow will introduce more complexity and what prevent a user to set the variable and get the same result?

Reply:
Except on mesh expansion - where everything is using environment variables from sidecar.env.

And except that this exposes an internal testing detail to the user.